### ISSUE TITLE
lots of plplot error messages when ploting after special "window" sequence

### ISSUE THREAD

**Issue by brandy125** [gnudatalanguage/gdl#1835]
I installed the latest version without problems in the usual way but now I get lots of error messages from plplot when executing the commands below in exactly that sequence:

```
bash$ gdl
  GDL - GNU Data Language, Version v1.0.5
- For basic information type HELP,/INFO
- Default library routine search path used (GDL_PATH/IDL_PATH env. var. not set): /usr/local/bin/../share/gnudatalanguage/lib
- Using WxWidgets as graphics library (windows and widgets).
- Using local drivers in /usr/local/bin/../share/gnudatalanguage/drivers
- Please report bugs, feature or help requests and patches at: https://github.com/gnudatalanguage/gdl

GDL> 
GDL> window,2
GDL> window,/free
GDL> window,2
GDL> wset,32
GDL> plot,findgen(10)

*** PLPLOT ERROR, ABORTING OPERATION ***
plvpor: Please call plinit first, aborting operation

*** PLPLOT ERROR, ABORTING OPERATION ***
plwind: Please set up viewport first, aborting operation

*** PLPLOT ERROR, ABORTING OPERATION ***
pleop: Please call plinit first, aborting operation

*** PLPLOT ERROR, ABORTING OPERATION ***
plbop: Please call plinit first, aborting operation

*** PLPLOT ERROR, ABORTING OPERATION ***
plvpor: Please call plinit first, aborting operation

*** PLPLOT ERROR, ABORTING OPERATION ***
plwind: Please set up viewport first, aborting operation

*** PLPLOT ERROR, ABORTING OPERATION ***
plcol1: Please call plinit first, aborting operation

*** PLPLOT ERROR, ABORTING OPERATION ***
plwind: Please set up viewport first, aborting operation

*** PLPLOT ERROR, ABORTING OPERATION ***
plstyl: Please call plinit first, aborting operation
...
```

**Comment by GillesDuvert** [comment]
@brandy125 could you pls comment out line 227 in gdlgstream.hpp? line says `plend1(); //strange. Was expected ...`

**Comment by brandy125** [comment]
Yes! Worked! 

Thanks!

**Comment by GillesDuvert** [comment]
regression (?? ?line was uncommented???) corrected (above) but implies an update of 1.0.5
